github.go

```diff
@@ -256,10 +269,27 @@ func (m *GithubClient) UpdateCommitStatus(commitRef, statusContext, status strin
 
 	// Format build page
 	build := os.Getenv("ATC_EXTERNAL_URL")
+	if m.TargetURL == "" {
+		if targetURL != "" {
+			build = targetURL
+		}
+	} else {
+		build = m.TargetURL
+	}
+
 	if build != "" {
 		build = strings.Join([]string{build, "builds", os.Getenv("BUILD_ID")}, "/")
 	}
 
+	desc := "Concourse CI build %s"
+	if m.Description == "" {
+		if description != "" {
+			desc = description + " %s"
```

Comment:
In this case (`put` description parameter) I think it makes sense to just use the description as is and not append the status, since the status is being set by the user in the same call.

Reply:
you're right. Solved.